<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mADeviceBrowser</class>
 <widget class="QDialog" name="mADeviceBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Device Browser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="miniAudicle.qrc">
    <normaloff>:/icon/miniAudicle.png</normaloff>:/icon/miniAudicle.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="margin" stdset="0">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,0">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="audioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Audio</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="midiButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>MIDI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hidButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>HID</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QComboBox" name="audioDriver">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;choose audio driver to the see corresponding audio devices&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="audioDriverLabel">
       <property name="text">
        <string>Audio driver</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="miniAudicle.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>audioButton</sender>
   <signal>clicked()</signal>
   <receiver>mADeviceBrowser</receiver>
   <slot>showAudio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>midiButton</sender>
   <signal>clicked()</signal>
   <receiver>mADeviceBrowser</receiver>
   <slot>showMIDI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hidButton</sender>
   <signal>clicked()</signal>
   <receiver>mADeviceBrowser</receiver>
   <slot>showHID()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audioDriver</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>mADeviceBrowser</receiver>
   <slot>selectedAudioDriverChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showAudio()</slot>
  <slot>showMIDI()</slot>
  <slot>showHID()</slot>
  <slot>selectedAudioDriverChanged()</slot>
 </slots>
</ui>
